packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -180,7 +180,7 @@ export async function runBuildAndSubmitAsync(
     buildCtxByPlatform[toAppPlatform(buildProfile.platform)] = buildCtx;
   }
 
-  if (flags.localBuildOptions.enable) {
+  if (flags.localBuildOptions.localBuildMode !== LocalBuildMode.DISABLED) {
```

Comment:
🤔🤔🤔 Shouldn't this be:
```suggestion
  if (flags.localBuildOptions.localBuildMode === LocalBuildMode.DISABLED) {
```

Reply:
If not, then this enum name is weird/confusing.